examples/plugin/CMakeLists.txt

```diff
@@ -5,7 +5,8 @@ set(OSVR_EXAMPLE_DEVICE_PLUGINS_SIMPLE
     com_osvr_example_DummyDetectAndCreateAsync
     com_osvr_example_EyeTracker
     com_osvr_example_Locomotion
-    com_osvr_example_MultipleAsync)
+    com_osvr_example_MultipleAsync
+    com_osvr_example_Tracker)
```

Comment:
So by building the plugin here, as opposed to separately down below, you're accepting the manual-load, no-install properties of all those other plugins. Just wanted to make sure that was intentional.